crds/objectstorage.k8s.io_bucketaccessclasses.yaml

```diff
@@ -36,8 +36,18 @@ spec:
             additionalProperties:
               type: string
             type: object
-          policyActions:
-            type: string
+          policyActionsConfigMap:
+            properties:
+              name:
+                description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                type: string
+              namespace:
+                description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+                type: string
+              uid:
```

Comment:
Is uid needed?

Reply:
not required but I am reusing the ObjectReference to not define one more type.